README.rst

```diff
@@ -50,14 +53,19 @@ libraries and CLI tools, you can use your package manager.
 
 .. code:: bash
 
-    $ sudo yum install antiword unrtf poppler-utils libjpeg-dev
+    $ sudo yum install antiword unrtf poppler-utils libjpeg-dev tesseract-ocr
 
 Or for debian-based systems:
 
 .. code:: bash
 
-    $ sudo apt-get install antiword unrtf poppler-utils libjpeg-dev
+    $ sudo apt-get install antiword unrtf poppler-utils libjpeg-dev tesseract-ocr
```

Comment:
I believe these two changes also belong in the other PR, as well as test.png below.

Reply:
You may wish to add this package to the `before_install` for Travis CI (in the other branch.)